acquia/acquia_cms
# ACMS-1711: do not import media if installed from profile.

**Motivation**
Fixes #ACMS-1711

**Proposed changes**
do not import media if installed from profile.

**Alternatives considered**
NA

**Testing steps**
- Checkout ACMS-1711 branch using `git clone --branch ACMS-1711 git@github.com:acquia/acquia_cms.git`
- Install all dependencies using `composer install && composer update --lock`
- Install site using `./vendor/bin/drush si minimal --account-pass=admin && drush in acquia_cms_image -y`
- Verify image media created at `/admin/content/media`. 
- Export config using `./vendor/bin/drush cex -y`
- Get site uuid using `./vendor/bin/drush cget system.site uuid`
- Copy the uuid
- **Site install then config import:**
- Install site using `./vendor/bin/drush si minimal --account-pass=admin && ./vendor/bin/drush cset system.site uuid <copied-uuid> -y && ./vendor/bin/drush cim -y && ./vendor/bin/drush cr`
- Verify image media created at `/admin/content/media`. 
- **Site install with existing site:**
- Install site using `./vendor/bin/drush si minimal --account-pass=admin --existing-config && drush cr`
- Verify image media created at `/admin/content/media`. 

**Merge requirements**
